backend/src/agents/web_agent.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from src.prompts import PromptEngine
+from .agent_types import Parameter
+from .agent import Agent, agent
+from .tool import tool
+from src.utils import Config
+from src.utils import update_scratchpad
+from src.utils.web_utils import search_urls, scrape_content, summarise_content
+from .validator_agent import ValidatorAgent
+
+logger = logging.getLogger(__name__)
+config = Config()
+
+engine = PromptEngine()
+
+
+def web_general_search_core(search_query, llm, model) -> list:
```

Comment:
This is returning a list, what is it a list of? String? perhaps we should be returning a json object or maybe a json string, that way it would be more easily manipulated.

Reply:
Changed the return type to JSON.